src/msw/dcmemory.cpp

```diff
@@ -131,6 +133,23 @@ void wxMemoryDCImpl::DoSelect( const wxBitmap& bitmap )
     {
         m_oldBitmap = hBmp;
     }
+
+    // Remember content scale factor used by the bitmap: we don't use it
+    // ourselves, but this can be needed later for creating fonts of the
+    // correct size.
+    m_contentScaleFactor = bitmap.GetScaleFactor();
+
+    // The font may need to be adjusted for the new scale factor.
+    SetFont(GetFont());
+}
+
+void wxMemoryDCImpl::SetFont(const wxFont& font)
```

Comment:
Are similar checks as in `wxMSWDCImpl::SetFont` useful/needed? E.g. to prevent creating copies of `wxFont`. Like:
```
    if ( font == m_font )
        return;

    if ( font.IsOk() )
    {
    }
`

Reply:
Never mind, this doesn't work, I tested it and `font == m_font` prevents the initial font from being adjusted to the correct DPI.

Reply:
Yes, exactly.